src/lib/snark_worker/functor.ml

```diff
@@ -1,10 +1,10 @@
 open Core
 open Async
 open Events
+open Snark_work_lib
 
 module Make = struct
   module Rpcs = Rpcs.Make
-  module Work = Snark_work_lib
```

Comment:
Just noticed, it's not a functor anymore 😆 

Suggest removing the `module Make = struct ... end` wrapping

And then rename `functor.ml` -> `worker_base.ml`.

Also, in `snark_worker.ml` it makes no sense to define `module Worker = ... end` followed by `include Worker`.